### PR TITLE
feat(@desktop/wallet): Networks - New List UX with link mainnet/testnet + testnet mode enable/disable

### PR DESCRIPTION
**Pull request by Khushboo-dev-cpp**
#11252 

status-go PR : https://github.com/status-im/status-go/pull/3697#pullrequestreview-1514291664

### What does the PR do

New networks settings view

### Affected areas

Wallet Settings, Networks list

### StatusQ checklist

- [ ] add documentation if necessary (new component, new feature)
- [ ] update sandbox app
  - in case of new component, add new component page
  - in case of new features, add variation to existing component page
  - nice to have: add it to the demo application as well
- [ ] test changes in both light and dark theme?

https://github.com/status-im/status-desktop/assets/60327365/9304e308-6d98-47dc-a128-380190617330

### Screenshot of functionality (including design for comparison)

- [x] I've checked the design and this PR matches it

https://github.com/status-im/status-desktop/assets/60327365/7a5ce1b0-c07a-40c2-9631-4abed0e7de4a

### CODE REVIEW

File: src/app_service/common/network_constants.nim [hunk 1]
```diff
@@ -110,22 +110,24 @@ var NETWORKS* = %* [
     "isTest": false,
     "layer": 1,
     "enabled": true,
+    "relatedChainId": 5,
   },
   {
     "chainId": 5,
-    "chainName": "Goerli",
+    "chainName": "Mainnet",
```
Comment (alaibe) [inline src/app_service/common/network_constants.nim:117]: Does that impact the network selector?
Also what about edit network?

Comment (Khushboo-dev-cpp) [reply src/app_service/common/network_constants.nim:117]: Edit network will be done in a following task. not covered here

About the network filter, I found an issue which I am going to fix, but expected to work after that!

**Review by stefandunca — APPROVED** [review]
Why do we keep duplicate network information? Can't all the metadata be moved in status-go? We seem to have duplicate code that has to be kept in sync.

File: src/app_service/common/network_constants.nim (hunk 1 above)
Comment (Khushboo-dev-cpp) [reply src/app_service/common/network_constants.nim:117]: Done @alaibe !

**Comment by Khushboo-dev-cpp** [comment]
> Why do we keep duplicate network information? Can't all the metadata be moved in status-go? We seem to have duplicate code that has to be kept in sync.

I am not sure about the thought process behind this @alaibe can you tell why this is needed?
anyway I would prefer to tackle this in a separate task

**Review by alaibe — APPROVED** [review]

**Comment by alaibe** [comment]
@stefandunca @Khushboo-dev-cpp yes we need to move all the networks to status-go but we haven't done it yet

**Comment by Khushboo-dev-cpp** [comment]
> @stefandunca @Khushboo-dev-cpp yes we need to move all the networks to status-go but we haven't done it yet

Created https://github.com/status-im/status-desktop/issues/11435

File: ui/imports/shared/popups/AlertPopup.qml
Comment (caybro) [inline ui/imports/shared/popups/AlertPopup.qml:1]: The file move will break MintTokenSettingsPanel.qml! You need to add the corresponding `import` there

**Review by caybro — CHANGES_REQUESTED** [review]

**Comment by Khushboo-dev-cpp** [comment]
> @Khushboo-dev-cpp small UI findings:
> 
> * [ ]  **Turn on testnet mode** -> per figma Testnet should start with capital T
> 
> Screenshot 2023-07-11 at 16 39 15
> * [ ]  Testnet banner at the top is missing `i` icon at the left and `is` should not be present in the text
> 
> Screenshot 2023-07-11 at 16 28 36
> image
> 
> * [ ]  **Turn off testnet mode** -> per figma **Testnet mode** in this case should be bold
> 
> Screenshot 2023-07-11 at 16 39 15

1. Comment from ben to keep it consistently small all over https://www.figma.com/file/idUoxN7OIW2Jpp3PMJ1Rl8?node-id=15264:344223&mode=dev#495140256
2. added missing icon
image
3. Made the testnet mode highlighted in popup text

image
image
@anastasiyaig Ive addressed your findings :)

File: ui/StatusQ/src/StatusQ/Controls/StatusBaseButton.qml [hunk 2]
```diff
@@ -19,7 +19,8 @@ Button {
     enum Type {
         Normal,
         Danger,
-        Primary
+        Primary,
+        Warning
```
[the remark this answers is not in the document]
Comment (Khushboo-dev-cpp) [reply ui/StatusQ/src/StatusQ/Controls/StatusBaseButton.qml:23]: Ive added it in the PR description above :)

**Comment by caybro** [comment]
> > @Khushboo-dev-cpp small UI findings:
> > 
> > * [ ]   **Turn on testnet mode** -> per figma Testnet should start with capital T
> > 
> > Screenshot 2023-07-11 at 16 39 15
> > * [ ]  Testnet banner at the top is missing `i` icon at the left and `is` should not be present in the text
> > Screenshot 2023-07-11 at 16 28 36
> > image
> > 
> > * [ ]   **Turn off testnet mode** -> per figma **Testnet mode** in this case should be bold
> > 
> > Screenshot 2023-07-11 at 16 39 15
> 
> 1. Comment from ben to keep it consistently small all over https://www.figma.com/file/idUoxN7OIW2Jpp3PMJ1Rl8?node-id=15264:344223&mode=dev#495140256
> 2. added missing icon
>    image
> 3. Made the testnet mode highlighted in popup text
> 
> image image @anastasiyaig Ive addressed your findings :)

I can read "tesnet" mode :) Typo :police_officer:

**Review by caybro — APPROVED** [review]
Nice, thanks for the fixes!

Modulo the "tesnet" typo ;)

**Comment by Khushboo-dev-cpp** [comment]
> Nice, thanks for the fixes!
> 
> Modulo the "tesnet" typo ;)

Fixed :)

**Comment by anastasiyaig** [comment]
thanks @Khushboo-dev-cpp please merge it !

**Review by anastasiyaig — APPROVED** [review]

**Pull request merged**